### PR TITLE
Merging Database information

### PR DESCRIPTION
**Pull request by savvyp**
- check the add_to_list mapping in class Record

### CODE REVIEW

File: SafeFoodie_Fresh/lib/screens/add_to_list.dart [hunk 1]
```diff
@@ -18,14 +18,14 @@ class _AddtoList extends State<AddtoList> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar:
-          AppBar(centerTitle: true, title: const Text('test test firebase work!!')),
+          AppBar(centerTitle: true, title: const Text('Lists')),
```
Comment (Arbanzo) [inline SafeFoodie_Fresh/lib/screens/add_to_list.dart:21]: good idea to change that haha

**Review by Arbanzo — APPROVED** [review]
Good work on this

**Review by JRuan18 — APPROVED** [review]

**Comment by Arbanzo** [comment]
Please just rebuild and commit create new to test, then  re-open this merge to avoid conflict

File: SafeFoodie_Fresh/lib/screens/add_to_list.dart (hunk 1 above)
Comment (savvyp) [reply SafeFoodie_Fresh/lib/screens/add_to_list.dart:21]: changed!

**Pull request closed without merging**